js/models/profile/Profile.js

```diff
@@ -13,7 +13,7 @@ export default class extends BaseModel {
       handle: '',
       location: '',
       moderator: false,
-      name: `ob ${Math.random().toString(36).slice(2)}`,
+      name: `OB ${Math.random().toString(36).slice(1).substring(1,7)}`,
```

Comment:
You don't need the slice and the substring, the slice is cutting off the first character (which is a 0), then the substring is cutting off the first character again (which is now a .). 

You could do the same thing with `OB ${Math.random().toString(36).substring(2,8)}`

There is a chance the random number won't have enough decimals (0.2, for instance), if we care about that we could do a little more here to fix it.